server/views/action.njk

```diff
@@ -0,0 +1,11 @@
+{% extends 'layout.njk' %}
+
+{% from "govuk/components/breadcrumbs/macro.njk" import govukBreadcrumbs %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+
+{% block content %}
+  <div class="govuk-body">
+    Action was {{ "successful" if actionResult.success else "unsuccessful" }}
```

Comment:
Bit of a nitpick, but the mocked endpoint doesn't return anything other than success, so the page will never show unsuccessful